ginga/tests/test_numpy.py

```diff
@@ -0,0 +1,63 @@
+
+import numpy as np
+
+from ginga import AstroImage, trcalc
+from ginga.misc import log
+
+
+class TestNumpy:
+    def setup_class(self):
+        self.logger = log.get_logger("TestNumpy", null=True)
+
+    def _get_data(self, shape, data_np=None):
+        if data_np is None:
+            data_np = np.random.randint(0, 10000, shape)
+        return data_np.reshape(shape)
+
+    def test_numpy_slice_trcalc(self):
+        """Test that we can get a subslice of an array.
+        """
+        arr_np = self._get_data((1000, 500))
+
+        x_slice, y_slice = slice(12, 499, 3), slice(10, 951, 11)
+        view = (y_slice, x_slice)
+        data_np = trcalc.fancy_index(arr_np, view)
+        assert isinstance(data_np, np.ndarray)
+        assert data_np.shape == (86, 163)
```

Comment:
Why not also compare the array values here?

Reply:
Ditto